plsync/mlabconfig.py

```diff
@@ -191,16 +191,43 @@ def export_mlab_zone_records(output, sites, experiments):
     export_experiment_records(output, sites, experiments)
 
 
+def get_revision(revision_path):
+    """Returns a two digit revision number as a string.
+
+    Everytime the same revision_path is provided, the previous revision number
+    is incremented by one and returned. However, the revision number does not
+    increase beyond "99".
+
+    Args:
+        revision_path: str, the full path to a temporary file to read previous
+            and store latest revision number.
+
+    Returns:
+        str, a two digit revision number, e.g. "00", "01", etc, up to "99".
+    """
+    n = 0
+    if os.path.exists(revision_path):
+        with open(revision_path) as f:
+            n = int(f.read())
+            if n < 99:
+                n += 1
+            else:
+                logging.error('Revision is too large to increase!')
+
+    revision = '%02d' % n
+    with open(revision_path, 'w') as f:
+        f.write(revision)
```

Comment:
This is not a huge problem, but what prevents old/stale revision files from piling up in /tmp? To prevent this, perhaps the file could have a static name, and the content could indicate the prefix and the revision number together? Something like this:

```
{
    'prefix': '20160929',
    'revision': '02'
}
```